twilight-http/src/response/future.rs

```diff
@@ -134,11 +134,7 @@ impl InFlight {
             #[cfg(feature = "decompression")]
```

Comment:
This module's structs store `guild_id`'s and has a `set_guild_id` method defined on `ResponseFuture`, all of these should be removed.